warp10/src/main/java/io/warp10/script/functions/SNAPSHOT.java

```diff
@@ -440,14 +440,14 @@ private static String processString(String s) {
     //
 
     while(idx < chars.length) {
-      if ("'".charAt(0) == chars[idx] || chars[idx] < 32) {
+      if ('\'' == chars[idx] || chars[idx] < ' ') {
 
         if (null == sb) {
```

Comment:
You don't need to allocate a new StringBuilder, simply accept one as parameter to processString